packages/web-ui/components/chat.tsx

```diff
@@ -224,7 +224,12 @@ export default function Chat({ chat, roomId, isOwner, ownerRoles }: ChatProps) {
 
   const sendMessage = (value: string) => {
     socket.emit('message', {
-      data: createSocketMessageRequestFactory(roomId, value, userId ?? ''),
+      data: createSocketMessageRequestFactory(
```

Comment:
this factory should be refactored to receive an object with these keys, it's starting to have too many parameters